src/CoreFoundation/CFMutableString.cs

```diff
@@ -0,0 +1,134 @@
+// Copyright 2019 Microsoft Corporation
+
+#if !COREBUILD
+
+using System;
+using System.Runtime.InteropServices;
+
+using Foundation;
+using ObjCRuntime;
+
+namespace CoreFoundation {
+	
+	public class CFMutableString : CFString {
+
+		public CFMutableString (IntPtr handle)
```

Comment:
`internal`? I know the CFString version is public, but that doesn't mean we have to add new public ones.

Reply:
changing to `protected` like the 2 argument `.ctor`

The swift bindings do not seems to make to limit subclassing this type - and I don't see why we should impose our own arbitrary limits (there's enough _non-arbitrary _ of them already)